polarityio/cyberchef
# Fixed Misc Bugs

## What's New:
- Fixed "Cannot read property 'type' of undefined" error
- Improved the "Ignore Entity Types" user option to cover edge cases with integration framework regex recognition.

## How to Test
- Change the package.json line 7 to be `"install": "VERSION=\"3.2.2-beta\" && wget -c \"https://github.com/polarityio/cyberchef/releases/download/${VERSION}/cyberchef-${VERSION}.tgz\" -O - | tar -xz -C /app/polarity-server/integrations/",` before you do `npm run install`
- Search `1a9804f0c374283b094e9e55dc5ee128` and make sure you don't get a response
